iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -384,11 +384,17 @@ export class Inspector {
                 .text(`${prefix} (${this._reportSet.prefixMap()[prefix]})`)
                 .appendTo('#search-filter-namespaces select');
         }
-        for (const targetDocument of this._reportSet.getTargetDocuments()) {
-            $("<option>")
-                .attr("value", targetDocument ?? ':default')
-                .text(targetDocument ?? `<${i18next.t("search.default")}>`)
-                .appendTo('#search-filter-target-document select');
+        const targetDocuments = Array.from(this._reportSet.getTargetDocuments());
+        if (targetDocuments.length == 1 && targetDocuments[0] == null) {
+            $('#search-filter-target-document').hide();
```

Comment:
Is it worth displaying the filter in the case where there's only a single target and it's not the default?

Reply:
I've remembered my thinking here.  We don't generally hide filters just because there's only a single option, and it's actually useful to show the filter because it makes it explicit that there is only one.

But, many environments don't support multi-target documents at all, and in this case users may well not even know what a target document is and this filter is just confusing clutter, so I think there's value in hiding it (at the cost of it no longer being explicitly clear that there is only one target).